### ISSUE TITLE
Finetune MViT-B 32x3 on Charades

### ISSUE THREAD

**Issue by Yui010206** [facebookresearch/SlowFast#556]
Hi teams, appreciate your excellent work! 
I'm trying to fine-tune the MViT-B 32x3 on Charades with this codebase.
However, I got a very low mAP result compared with the reported one, although I followed the setting you mentioned in supplementary D.3. And I'm also confused about the statement "A per-class sigmoid output is used to account for the multi-class nature.".  I believe there must be some configs/codes missing in my training.
Will it be convenient for your group to share the fine-tune config or related code with me (shoubin@cs.unc.com)? It will be greatly helpful for our undergoing research. Thanks in advance!
